### PR TITLE
GA plugin args support

### PR DESCRIPTION
**Pull request by Bvz2001**
In [analytics.js documentation](https://developers.google.com/analytics/devguides/collection/analyticsjs/command-queue-reference#require) `require` command takes `pluginName` and optionally `pluginOptions`:
`ga('[trackerName.]require', pluginName, [pluginOptions]);`
while react-ga supports only `pluginName` :(

Added support for `pluginOptions`.

### CODE REVIEW

**Comment by yangshun** [comment]
hey @Bvz2001 , nice! Could you add some tests too?

File: src/index.js [hunk 1]
```diff
@@ -302,14 +302,15 @@ var reactGA = {
...
-    require: function(name) {
+    require: function(name, options) {
       if (typeof ga === 'function') {
-        ga('require', name);
+        ga('require', name, options);
       }
 
       if (_debug) {
-        log('called ga(\'require\', \'' + name + '\');');
+        log('called ga(\'require\', \'' + name + '\', ' + JSON.stringify(options) + ');');
```
Comment (yangshun) [inline src/index.js]: Should check whether `options` exist.

File: README.md [hunk 2]
```diff
@@ -168,15 +168,20 @@ ga.exception( { description: 'An error ocurred', fatal: true } );
 |args.fatal|`String`. Optional. Set to `true` if it was a fatal exception.|
 
 
-#### ga.plugin.require(name)
+#### ga.plugin.require(name, options)
```
Comment (yangshun) [inline README.md]: `#### ga.plugin.require(name, [options])`

**Comment by impronunciable** [comment]
There are conflicts because we changed the linting, sorry. Please fix that conflicts, add tests and it should be good to go.

**Comment by Bvz2001** [comment]
@yangshun it seems I'm ready to go ;)

File: src/index.js [hunk 3]
```diff
@@ -303,14 +303,48 @@ var reactGA = {
...
-      if (_debug) {
-        log('called ga(\'require\', \'' + name + '\');');
+        // Required Fields
+        if (!name) {
+          warn('name is required in .require()');
+          return;
+        }
+
+        name = trim(name);
+        if (path === '') {
```
Comment (impronunciable) [inline src/index.js]: Tests not passing because of this error. Path does not exist. Please fix this and I think we are done (also fix the warn message.)

**Comment by Bvz2001** [comment]
@impronunciable tests and code style fixed

**Comment by impronunciable** [comment]
@Bvz2001 just one jscs error and ready to go

```
Illegal trailing whitespace at /Users/dan/Projects/react-ga/src/index.js :
   345 |            log('called ga(\'require\', \'' + name + '\');');
   346 |          }
   347 |
------------------^
   348 |          return;
   349 |        }
```

**Comment by impronunciable** [comment]
Merged. Thank you @Bvz2001

**Pull request merged**